deploy/crds/hco00.crd.yaml

```diff
@@ -45,6 +45,42 @@ spec:
           spec:
             description: HyperConvergedSpec defines the desired state of HyperConverged
             properties:
+              certConfig:
+                description: certConfig holds the rotation policy for internal, self-signed
+                  certificates
+                properties:
+                  ca:
+                    description: CA configuration - CA certs are kept in the CA bundle
+                      as long as they are valid
+                    properties:
+                      duration:
+                        description: The requested 'duration' (i.e. lifetime) of the
+                          Certificate. This should comply with golang's ParseDuration
+                          format (https://golang.org/pkg/time/#ParseDuration)
+                        type: string
+                      renewBefore:
+                        description: The amount of time before the currently issued
+                          certificate's `notAfter` time that we will begin to attempt
+                          to renew the certificate. This should comply with golang's
+                          ParseDuration format (https://golang.org/pkg/time/#ParseDuration)
+                        type: string
+                    type: object
+                  server:
+                    description: Server configuration - Certs are rotated and discarded
+                    properties:
+                      duration:
+                        description: The requested 'duration' (i.e. lifetime) of the
+                          Certificate. This should comply with golang's ParseDuration
+                          format (https://golang.org/pkg/time/#ParseDuration)
+                        type: string
+                      renewBefore:
+                        description: The amount of time before the currently issued
+                          certificate's `notAfter` time that we will begin to attempt
+                          to renew the certificate. This should comply with golang's
+                          ParseDuration format (https://golang.org/pkg/time/#ParseDuration)
+                        type: string
+                    type: object
+                type: object
```

Comment:
I'd expect to explicitly see a `default: {...}` here

Reply:
done